test/hetzner_floating_ip_test.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+
+set -eu -o pipefail
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+# testing parameters
+dev=`ip link show | grep -B1 ether | cut -d ":" -f2 | head -n1 | cut -d " " -f2`
+vip=192.168.1.12
```

Comment:
don't think we need that here :)

Reply:
Originally I've put this in place since `ip`, `netmask` and `interface` must be configured/passed correctly for vip-manager to work, since it will query the network interface on startup.
I worked around this in commit 629c1bd4c235d4ca7c201cb1440d11b481cbc865, hope this solution works for you too.